build.gradle

```diff
@@ -111,6 +109,20 @@ static Map<String, String> getEnvironmentVariables(String... envNames) {
     return envNames.collectEntries {envName -> [(envName): System.getenv(envName)] }
 }
 
+String getArtifactoryForPublication() {
+    def url = "https://software.r3.com/artifactory/conclave-maven"
+    def conclaveGraal = getConclaveGraalVersion()
+    if (conclaveGraal.endsWith("-SNAPSHOT")) {
```

Comment:
Few things here:
- I guess we should somehow agree on some formatting in the code, i.e. I would personally put a line before the if and remove it after the code below. Not a blocker though.
- I would make the condition below in fewer lines, not a blocker, but nicer to see
```
def conclaveGraal = getConclaveGraalVersion()
def repositoryPath = if (conclaveGraal.endsWith("-SNAPSHOT")) ? "conclave-maven-dev" : "conclave-maven-release"
return "https://software.r3.com/artifactory/" + repositoryPath
```
Or similar